### PR TITLE
P/brent/gsum

### LINKED ISSUE #1024: Create real-time Gsum channel
Create real-time gsum channel to compliment existing IMU channels.

Channel name shall be Gsum; units; 'G' and default max G's of 3 (range 0-3G)

### PR DESCRIPTION
**Pull request by brentpicasso**
Create real-time Gsum chanel. resolves #1024

### CODE REVIEW

File: include/logger/loggerConfig.h [hunk 1]
```diff
@@ -94,7 +94,7 @@ struct TimeConfig {
 
 #define EMPTY_CHANNEL_CONFIG {"","", 0.0f, 0.0f, SAMPLE_DISABLED, 0}
 
-/*
+/*gsyn
```
Comment (ddimensia) [inline include/logger/loggerConfig.h]: ?

Comment (brentpicasso) [reply include/logger/loggerConfig.h]: whoops! fixed.

File: CHANGELOG [hunk 2]
```diff
@@ -1,3 +1,5 @@
+= 2.16.0=
+
```
Comment (stieg) [inline CHANGELOG]: LoL... nothing has changed this release!

File: src/logger/loggerSampleData.c [hunk 3]
```diff
@@ -220,6 +221,14 @@ float get_imu_sample(int channelId)
         float value = imu_read_value(channelId, c);
         return value;
 }
+
+float get_imu_gsum_getter(void)
+{
+        LoggerConfig *config = getWorkingLoggerConfig();
+        float x_value = imu_read_value(IMU_CHANNEL_X, &config->ImuConfigs[IMU_CHANNEL_X]);
+        float y_value = imu_read_value(IMU_CHANNEL_Y, &config->ImuConfigs[IMU_CHANNEL_Y]);
```
Comment (stieg) [inline src/logger/loggerSampleData.c]: So what about Z axis?  We do live in a 3D world after-all.

Comment (brentpicasso) [reply src/logger/loggerSampleData.c]: I considered this - but it's not an industry standard thing to mix in the Z axis. I think bumps and lifts would mix in undesired noise into Gsum.

File: CHANGELOG (hunk 2 above)
Comment (brentpicasso) [reply CHANGELOG]: Ha, ok added something.

**Pull request merged**